package.json

```diff
@@ -54,7 +54,7 @@
 		"@typescript-eslint/parser": "8.38.0",
 		"@viamrobotics/prime-core": "0.1.5",
 		"@viamrobotics/sdk": "0.46.0",
-		"@viamrobotics/svelte-sdk": "0.4.4",
+		"@viamrobotics/svelte-sdk": "0.4.3",
```

Comment:
The newest version doesn’t have the the throw on error prop if you want to try bumping up instead.